devicetypes/smartthings/zll-rgbw-bulb.src/zll-rgbw-bulb.groovy

```diff
@@ -228,7 +228,8 @@ def setColorTemperature(value) {
 }
 
 def setLevel(value, rate = null) {
-	zigbee.setLevel(value) + zigbee.onOffRefresh() + zigbee.levelRefresh() //adding refresh because of ZLL bulb not conforming to send-me-a-report
+	//adding refresh because of ZLL bulb not conforming to send-me-a-report
+	zigbee.setLevel(value) + (value?.toInteger() > 0 ? zigbee.on() : []) + zigbee.onOffRefresh() + zigbee.levelRefresh()
```

Comment:
I added more comments on the ticket but this won't work because this command runs locally on the hub